contracts/contract_state_tree.md

```diff
@@ -59,4 +59,6 @@ The calls state tree contains all calls done in the block and their
 return values. The calls state tree is pruned in each block so that
 the latest block only contains the calls in that block.
 
+The key is `<contract_address><caller_address><caller_nonce><contract_address>`.
```

Comment:
Is this expected to contain contract address twice?

Reply:
See https://github.com/aeternity/epoch/blob/c8050fea2b033b9448d1e907befcaccfa17b24ac/apps/aecontract/src/aect_call_state_tree.erl#L56-L58